target/linux/ath79/dts/qca9563_tplink_tl-wa1201-v2.dts

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca956x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,tl-wa1201-v2", "qca,qca9563";
+	model = "TP-Link TL-WA1201 v2";
+
+	aliases {
+		label-mac-device = &eth0;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		mdio-gpio0 = &mdio2;
```

Comment:
Do we (still) need this?

Reply:
I'm not very well versed with dts files.

This particular alias is inspired by `qca9558_tplink_rex5x.dtsi` which has a similar setup (ar8031 phy with mdio over gpio bitbanged).
The [linux manual](https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/net/mdio-gpio.txt) regarding this states that such an alias is required.